cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -416,7 +437,46 @@ public void getApplicationDetails(FullHttpRequest request, HttpResponder respond
   }
 
   /**
-   * Decodes request coming from the {@link #getApplicationDetails(FullHttpRequest, HttpResponder, String)} call.
+   * Get {@link ApplicationDetail} for a given application
+   *
+   * @param request     {@link HttpRequest}
+   * @param responder   {@link HttpResponse}
+   * @param namespace   the namespace to get all application details   *
```

Comment:
extra '*' at the end of the line. Same comment in other places in this file

Reply:
Removed extra "*".

Fixing the param doc alignment